test/pub_test.dart

```diff
@@ -149,7 +139,7 @@ dev_dependencies:
 PSDependency findDependency(PSDependencyList deps, {String name}) =>
     deps.firstWhere((dep) => dep.name.text == name, orElse: () => null);
 
-testDepListContains(
+void testDepListContains(
```

Comment:
It's 100% our coding style across almost all of our code.

The only time you omit the return type is if you want dynamic implied. Period.